ajax/libs/findAndReplaceDOMText/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "findAndReplaceDOMText",
+  "filename": "./src/findAndReplaceDOMText.min.js",
+  "description": "findAndReplaceDOMText: DOM find/replace utility",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/padolsey/findAndReplaceDOMText.git"
+  },
+  "author": "James Padolsey",
+  "keywords": [
+    "DOM",
+    "find",
+    "replace"
+  ],
+  "license": "unlicense",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/padolsey/findAndReplaceDOMText.git",
+    "fileMap": [
+      {
+        "basePath": "src",
+        "files": [
+          "**/*"
```

Comment:
We only use `**/*` for `build` or `dist` folder, please use `findAndReplaceDOMText*` in this case, thanks!